en/ros2/px4_sdk.md

```diff
@@ -0,0 +1,336 @@
+# PX4 SDK
+
+The [PX4 SDK](https://github.com/PX4/px4_sdk) is a C++ library to simplify controlling PX4 from ROS 2.
+It allows to write external modes that are dynamically registered with PX4 and behave the same way as internal ones.
+A mode can send different types of setpoints, ranging from high-level navigation tasks all the way down to direct actuator controls.
+
+## Overview
+
+This diagram provides a conceptual overview:
+
+![ROS2 modes overview diagram](../../assets/middleware/ros2/px4_sdk/ros2_modes_overview.svg)
+
+<!-- Source: https://docs.google.com/drawings/d/1WByCfgcytnaow7r41VhYJL8OGrw1RjFO51GoPMQBCNA/edit -->
+
+The following sections define and explain the terms used in the diagram.
+
+### Definitions
+
+#### Mode
+
```

Comment:
This might be where you could say something like "A mode running in ROS2 is a first class peer of other modes in the system ..."